src/components/FeedbackButton.tsx

```diff
@@ -0,0 +1,105 @@
+import { EditIcon } from "@chakra-ui/icons";
+
+import {
+    Modal,
+    ModalOverlay,
+    ModalContent,
+    ModalHeader,
+    ModalFooter,
+    ModalBody,
+    ModalCloseButton,
+    Button,
+    useDisclosure,
+    IconButton,
+    Text,
+    Textarea
+} from "@chakra-ui/react";
+
+import React from "react";
+
+import { submitFeedback } from "../services/feedbackService";
+
+export function FeedbackButton() {
+    const { isOpen, onOpen, onClose } = useDisclosure();
+    const [feedbackString, setFeedbackString] = React.useState<string>("");
+    const [isFeedbackSubmit, setIsFeedbackSubmit] = React.useState<boolean>(false);
```

Comment:
super nit: i think this is better named "isFeedbackSubmitted", but i don't care strongly about this :)